### PR TITLE
W-17017644-hyperforce-prework-duke

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/_partials/include-nav-entities.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+//
+// tag::anypoint-regions-hyperforce[]
+https://jp1.platform.mulesoft.com/monitoring[Japan (CA)] or https://ca1.platform.mulesoft.com/monitoring[Canada (CA)]
+// end::anypoint-regions-hyperforce[]
+//
+
+//
+// tag::anypoint-regions-useu[]
+https://jp1.platform.mulesoft.com/monitoring[US] and https://ca1.platform.mulesoft.com/monitoring[EU]
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/include-nav-entities.adoc]:
```suggestion
https://anypoint.mulesoft.com/monitoring[US] and https://eu1.anypoint.mulesoft.com/monitoring[EU]
```

File: modules/ROOT/pages/_partials/include-nav-entities.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//
+// tag::open-insights[]
+. In the Anypoint Monitoring navigation menu, click *Insights* to open *Anypoints Insights*.
+// end::open-insights[]
+//
+
+//
+// tag::anypoint-regions-hyperforce[]
+https://jp1.platform.mulesoft.com/monitoring[Japan (CA)] or https://ca1.platform.mulesoft.com/monitoring[Canada (CA)]
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/include-nav-entities.adoc]:
```suggestion
https://jp1.platform.mulesoft.com/monitoring[Japan (JP)] or https://ca1.platform.mulesoft.com/monitoring[Canada (CA)]
```

File: modules/ROOT/pages/_partials/include-nav-entities.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
+//
+// tag::open-insights[]
+. In the Anypoint Monitoring navigation menu, click *Insights* to open *Anypoints Insights*.
+// end::open-insights[]
+//
+
+//
+// tag::anypoint-regions-hyperforce[]
+https://jp1.platform.mulesoft.com/monitoring[Japan (JP)] or https://ca1.platform.mulesoft.com/monitoring[Canada (CA)]
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/include-nav-entities.adoc]:
```suggestion
https://jp1.platform.mulesoft.com/monitoring[Japan (JP)] and https://ca1.platform.mulesoft.com/monitoring[Canada (CA)]
```

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,3 +1,21 @@
 {
-    "asciidoc.antora.enableAntoraSupport": false
```
Comment (dukesphere) [inline .vscode/settings.json:2]: repo level thing, not doc content

File: modules/ROOT/pages/_partials/include-monitoring-metrics-collection.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
 = How Anypoint Monitoring Collects Metrics
 
-The Anypoint Monitoring agent collects metrics and aggregates them on a per-minute basis, so a minute is the shortest possible time increment you can use in dashboards. Metrics collection begins at a random second of that minute, and collection repeats at the same second in subsequent minutes. For example, if the agent collects metrics at 12:00:15, it collects them again at 12:01:15, 12:02:15, and so on.
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/include-monitoring-metrics-collection.adoc:3]: just the dark green

**Review by tasmoore — APPROVED** [review]

**Pull request merged**